src/main/java/com/thealgorithms/conversions/AffineConverter.java

```diff
@@ -0,0 +1,23 @@
+package com.thealgorithms.conversions;
+
+public final class AffineConverter {
+    final double slope;
```

Comment:
What do you think about making them `private`?

Or these values ​​should be used in the future for some cases, such as `package private`?

Reply:
Ups. You are right.